msgid ""
msgstr "Project-Id-Version: pyload\n"
"Report-Msgid-Bugs-To: 'bugs@pyload.org'\n"
"POT-Creation-Date: 2014-04-08 15:01+0200\n"
"PO-Revision-Date: 2014-04-09 05:38-0400\n"
"Last-Translator: pyloadTeam <team@pyload.org>\n"
"Language-Team: Catalan\n"
"Language: ca_ES\n"
"MIME-Version: 1.0\n"
"Content-Type: text/plain; charset=UTF-8\n"
"Content-Transfer-Encoding: 8bit\n"
"Plural-Forms: nplurals=2; plural=(n != 1);\n"
"X-Generator: crowdin.net\n"

#: module/Core.py:182
msgid "Received Quit signal"
msgstr "S'ha rebut la senyal de Sortida"

#: module/Core.py:303
#, python-format
msgid "pyLoad already running with pid %s"
msgstr "ja s'estat executant pyLoad amb el pid %s"

#: module/Core.py:317
#, python-format
msgid "Failed changing group: %s"
msgstr "Ha fallat el canvi de el grup: %s"

#: module/Core.py:327
#, python-format
msgid "Failed changing user: %s"
msgstr "Ha fallat el canvi d'usuari: %s"

#: module/Core.py:329
msgid "folder for logs"
msgstr "carpeta de registre"

#: module/Core.py:340
msgid "Starting"
msgstr "Començant"

#: module/Core.py:341
#, python-format
msgid "Using home directory: %s"
msgstr "Emprant el directori d'inici: %s"

#: module/Core.py:350
msgid "pycrypto to decode container files"
msgstr "pycrypto descodificarà el contenidor de fitxers"

#: module/Core.py:353
msgid "folder for temporary files"
msgstr "carpeta per fitxers temporals"

#: module/Core.py:358
msgid "folder for downloads"
msgstr "carpeta de descàrregues"

#: module/Core.py:361
msgid "OpenSSL for secure connection"
msgstr "OpenSSL per connexions segures"

#: module/Core.py:365
msgid "Moving old user config to DB"
msgstr "Movent l'antiga configuració d'usuari a la DB"

#: module/Core.py:368
msgid "Please check your logindata with ./pyload.py -u"
msgstr "Comprova les teves dades de login amb ./pyload.py -u"

#: module/Core.py:371
msgid "All links removed"
msgstr "Tots els enllaços eliminats"

#: module/Core.py:402
#, python-format
msgid "Downloadtime: %s"
msgstr "Temps de descàrrega: %s"

#: module/Core.py:412
#, python-format
msgid "Free space: %s"
msgstr "Espai lliure: %s"

#: module/Core.py:432
msgid "Activating Accounts..."
msgstr "Activant Comptes..."

#: module/Core.py:438
msgid "Activating Plugins..."
msgstr "Activant Plugins..."

#: module/Core.py:441
msgid "pyLoad is up and running"
msgstr "pyLoad està en funcionament"

#: module/Core.py:460
msgid "restarting pyLoad"
msgstr "reiniciant pyLoad"

#: module/Core.py:464
msgid "pyLoad quits"
msgstr "pyLoad està sortint"

#: module/Core.py:521
#, python-format
msgid "Install %s"
msgstr "Instal·la %s"

#: module/Core.py:557
#, python-format
msgid "could not find %(desc)s: %(name)s"
msgstr "no s'ha pogut trobar %(desc)s: %(name)s"

#: module/Core.py:559
#, python-format
msgid "could not create %(desc)s: %(name)s"
msgstr "no s'ha pogut crear %(desc)s: %(name)s"

#: module/Core.py:580
msgid "shutting down..."
msgstr "apagant..."

#: module/Core.py:597
msgid "error while shutting down"
msgstr "ha succeït un error mentre s'apagava"

#: module/Core.py:661
msgid "killed pyLoad from Terminal"
msgstr "matat pyLoad des de el Terminal"

#: module/database/DatabaseBackend.py:174
msgid "Filedatabase was deleted due to incompatible version."
msgstr "L'arxiu de la base de dades ha estat eliminat per una incompatiblitat de la versió."

#: module/database/DatabaseBackend.py:189
msgid "Filedatabase could NOT be converted."
msgstr "El fitxer de la base de dades NO pot ser convertit."

#: module/database/DatabaseBackend.py:198
msgid "Database was converted from v2 to v3."
msgstr "La base de dades ha estat convertida de la v2 a la v3."

#: module/database/DatabaseBackend.py:206
msgid "Database was converted from v3 to v4."
msgstr "La base de dades ha estat convertida de la v3 a la v4."

#: module/database/DatabaseBackend.py:252
msgid "Converting old Django DB"
msgstr "Convertint l'antiga BD de Django"

#: module/database/FileDatabase.py:45
msgid "finished"
msgstr "finalitzat"

#: module/database/FileDatabase.py:45
msgid "offline"
msgstr "fora de línia"

#: module/database/FileDatabase.py:45
msgid "online"
msgstr "en línia"

#: module/database/FileDatabase.py:45
msgid "queued"
msgstr "en cua"

#: module/database/FileDatabase.py:45
msgid "skipped"
msgstr "saltat"

#: module/database/FileDatabase.py:45
msgid "waiting"
msgstr "esperant"

#: module/database/FileDatabase.py:45
msgid "temp. offline"
msgstr "temporalment fora de línia"

#: module/database/FileDatabase.py:45
msgid "starting"
msgstr "començant"

#: module/database/FileDatabase.py:45
msgid "failed"
msgstr "fallit"

#: module/database/FileDatabase.py:45
msgid "aborted"
msgstr "avortat"

#: module/database/FileDatabase.py:45
msgid "decrypting"
msgstr "desxifrant"

#: module/database/FileDatabase.py:45
msgid "custom"
msgstr "personalitzat"

#: module/database/FileDatabase.py:45
msgid "downloading"
msgstr "descarregant"

#: module/database/FileDatabase.py:45
msgid "processing"
msgstr "processant"

#: module/database/FileDatabase.py:45
msgid "unknown"
msgstr "deconegut"

#: module/database/FileDatabase.py:531 module/plugins/addon/IRCInterface.py:74
#: module/plugins/addon/XMPPInterface.py:83
#, python-format
msgid "Package finished: %s"
msgstr "Paquet finalitzat: %s"

#: module/remote/ThriftBackend.py:39
msgid "Using SSL ThriftBackend"
msgstr "Emprant SSL ThriftBackend"

#: module/manager/RemoteManager.py:35
#, python-format
msgid "Remote backend error: %s"
msgstr "Error al backend remot: %s"

#: module/manager/RemoteManager.py:82
#, python-format
msgid "Starting %(name)s: %(addr)s:%(port)s"
msgstr "Iniciant %(name)s: %(addr)s:%(port)s"

#: module/manager/RemoteManager.py:84
#, python-format
msgid "Failed loading backend %(name)s | %(error)s"
msgstr "S'ha fallat la carrega del backend %(name)s | %(error)s"

#: module/web/json_app.py:57
#, python-format
msgid "waiting %s"
msgstr "esperant %s"

#: module/thread/ServerThread.py:35
msgid "SSL certificates not found."
msgstr "No s'han trobat els certificats SSL."

#: module/thread/ServerThread.py:39
#, python-format
msgid "Sorry, we dropped support for starting %s directly within pyLoad"
msgstr "Ho sent molt, ja no permetem iniciar %s directament emprant pyLoad"

#: module/thread/ServerThread.py:40
msgid "You can use the threaded server which offers good performance and ssl,"
msgstr "Pots emprar el servidor amb fils que ofereix millor rendiment i ssl,"

#: module/thread/ServerThread.py:41
#, python-format
msgid "of course you can still use your existing %s with pyLoads fastcgi server"
msgstr "per suposat encara pots emprar el teu %s amb el servidor fastcgi de pyLoad"

#: module/thread/ServerThread.py:42
msgid "sample configs are located in the module/web/servers directory"
msgstr "les configuracions d'exemple estan al directori module/web/servers"

#: module/thread/ServerThread.py:49
#, python-format
msgid "Can't use %(server)s, python-flup is not installed!"
msgstr "No s'ha pogut emprar %(server)s, python-flup no està instal·lat!"

#: module/thread/ServerThread.py:56
#, python-format
msgid "Error importing lightweight server: %s"
msgstr "Error important el servidor lleuger: %s"

#: module/thread/ServerThread.py:57
msgid "You need to download and compile bjoern, https://github.com/jonashaag/bjoern"
msgstr "Has de descarregar i compilar el bjoern. https://github.com/jonashaag/bjoern"

#: module/thread/ServerThread.py:58
msgid "Copy the boern.so to module/lib folder or use Setup.py install"
msgstr "Copia el fitxer boern.so a la carpeta module/lib o empra Setup.py install"

#: module/thread/ServerThread.py:59
msgid "Of course you need to be familiar with linux and know how to compile software"
msgstr "Per suposat hauràs d'estar familiaritzat amb GNU/Linux i sabre com compilar programes"

#: module/thread/ServerThread.py:63
msgid "Server set to threaded, due to known performance problems on windows."
msgstr "El servidor està en mode multi-fil degut als problemes de rendiment a windows."

#: module/thread/ServerThread.py:80 module/thread/ServerThread.py:103
msgid "This server offers no SSL, please consider using threaded instead"
msgstr "Aquest servidor no ofereix SSL, si us plau considera l'opció d'emprar el servidor amb fils"

#: module/thread/ServerThread.py:82
#, python-format
msgid "Starting builtin webserver: %(host)s:%(port)d"
msgstr "Iniciant el servidor incorporat: %(host)s:%(port)d"

#: module/thread/ServerThread.py:87
#, python-format
msgid "Starting threaded SSL webserver: %(host)s:%(port)d"
msgstr "Iniciant el servidor web SSL amb fils: %(host)s:%(port)d"

#: module/thread/ServerThread.py:91
#, python-format
msgid "Starting threaded webserver: %(host)s:%(port)d"
msgstr "Iniciant el servidor web amb fils: %(host)s:%(port)d"

#: module/thread/ServerThread.py:97
#, python-format
msgid "Starting fastcgi server: %(host)s:%(port)d"
msgstr "Iniciant el servidor fastcgi: %(host)s:%(port)d"

#: module/thread/ServerThread.py:105
#, python-format
msgid "Starting lightweight webserver (bjoern): %(host)s:%(port)d"
msgstr "Iniciant el servidor web lleuger (bjoern): %(host)s:%(port)d"

#: module/web/pyload_app.py:125
msgid "You dont have permission to access this page."
msgstr "No tens permisos per accedir a aquesta pàgina."

#: module/web/pyload_app.py:193
msgid "Download directory not found."
msgstr "No s'ha trobat el directori de descàrregues."

#: module/web/pyload_app.py:260 module/web/pyload_app.py:267
msgid "unlimited"
msgstr "il·limitat"

#: module/web/pyload_app.py:262 module/web/pyload_app.py:269
msgid "not available"
msgstr "no disponible"

#: module/web/pyload_app.py:508
msgid "Run pyload.py -s to access the setup."
msgstr "Executa pyload.py -s per accedir a l'instal·lació."

#: module/network/HTTPDownload.py:245
#, python-format
msgid "Download chunks failed, fallback to single connection | %s"
msgstr "Descàrrega per trossos fallida, tornant a la connexió única | %s"

#: module/thread/PluginThread.py:183
#, python-format
msgid "Download starts: %s"
msgstr "Iniciant descàrrega: %s"

#: module/thread/PluginThread.py:189
#, python-format
msgid "Download finished: %s"
msgstr "Descàrrega finalitzada: %s"

#: module/thread/PluginThread.py:194 module/thread/PluginThread.py:365
#, python-format
msgid "Plugin %s is missing a function."
msgstr "El plugin %s troba a faltar una funció."

#: module/thread/PluginThread.py:202 module/thread/PluginThread.py:265
#: module/thread/PluginThread.py:382
#, python-format
msgid "Download aborted: %s"
msgstr "Descàrrega avortada: %s"

#: module/thread/PluginThread.py:222
#, python-format
msgid "Download restarted: %(name)s | %(msg)s"
msgstr "Descàrrega reiniciada: %(name)s | %(msg)s"

#: module/thread/PluginThread.py:231 module/thread/PluginThread.py:373
#, python-format
msgid "Download is offline: %s"
msgstr "La descàrrega està fora de línia: %s"

#: module/thread/PluginThread.py:234
#, python-format
msgid "Download is temporary offline: %s"
msgstr "La descàrrega està temporalment fora de línia: %s"

#: module/thread/PluginThread.py:237 module/thread/PluginThread.py:304
#, python-format
msgid "Download failed: %(name)s | %(msg)s"
msgstr "Descàrrega fallida: %(name)s | %(msg)s"

#: module/thread/PluginThread.py:254
msgid "Couldn't connect to host or connection reset, waiting 1 minute and retry."
msgstr "No s'ha pogut connectar amb el servidor o la connexió s'ha reiniciat, esperant 1 minuts per tornar-ho a provar."

#: module/thread/PluginThread.py:290
#, python-format
msgid "Download skipped: %(name)s due to %(plugin)s"
msgstr "Descàrrega omitida: %(name)s due to %(plugin)s"

#: module/thread/PluginThread.py:361
#, python-format
msgid "Decrypting starts: %s"
msgstr "Desxifrant comença en: %s"

#: module/thread/PluginThread.py:376 module/thread/PluginThread.py:394
#, python-format
msgid "Decrypting failed: %(name)s | %(msg)s"
msgstr "Desxifrat fallit: %(name)s | %(msg)s"

#: module/thread/PluginThread.py:388
#, python-format
msgid "Retrying %s"
msgstr "Re-intentant %s"

#: module/thread/PluginThread.py:635
#, python-format
msgid "Info Fetching for %(name)s failed | %(err)s"
msgstr "Obtenint informació per %(name)s failed | %(err)s"

#: module/manager/AddonManager.py:90 module/plugins/Addon.py:102
#, python-format
msgid "Error executing addon: %s"
msgstr "Error executant ganxos: %s"

#: module/manager/AddonManager.py:140
#, python-format
msgid "Failed activating %(name)s"
msgstr "L'activació de %(name)s ha fallat"

#: module/manager/AddonManager.py:144
#, python-format
msgid "Activated plugins: %s"
msgstr "Plugins activats: %s"

#: module/manager/AddonManager.py:145
#, python-format
msgid "Deactivate plugins: %s"
msgstr "Plugins desactivats: %s"

#: module/manager/ThreadManager.py:137
#, python-format
msgid "Reconnect Failed: %s"
msgstr "Reconnexió fallida: %s"

#: module/manager/ThreadManager.py:176
msgid "Reconnect script not found!"
msgstr "No s'ha trobat l'script de reconnexió!"

#: module/manager/ThreadManager.py:182
msgid "Starting reconnect"
msgstr "Iniciant reconnexió"

#: module/manager/ThreadManager.py:196
msgid "Failed executing reconnect script!"
msgstr "L'execució de l'script de reconnexió ha fallat!"

#: module/manager/ThreadManager.py:208
#, python-format
msgid "Reconnected, new IP: %s"
msgstr "Reconnectat, nova IP: %s"

#: module/manager/ThreadManager.py:288
msgid "Not enough space left on device"
msgstr "No queda suficient espai lliure al dispositiu"

#: module/plugins/Account.py:85 module/plugins/Account.py:91
#, python-format
msgid "Could not login with account %(user)s | %(msg)s"
msgstr "No s'ha pogut iniciar sessió amb el compte %(user)s | %(msg)s"

#: module/plugins/Account.py:86
msgid "Wrong Password"
msgstr "Contrasenya Errònia"

#: module/plugins/Account.py:240
#, python-format
msgid "Your Time %s has wrong format, use: 1:22-3:44"
msgstr "L'hora %s està en un format incorrecte, empra: 1:22-3:44"

#: module/plugins/Account.py:266
#, python-format
msgid "Account %s has not enough traffic, checking again in 30min"
msgstr "El compte %s no te suficient tràfic, es tornarà a comprovar d'aquí 30min"

#: module/plugins/Account.py:273
#, python-format
msgid "Account %s is expired, checking again in 1h"
msgstr "El compte %s està caducat, es tornarà a comprovar d'aquí 1h"

#: module/plugins/crypter/SerienjunkiesOrg.py:128
msgid "Downloadlimit reached"
msgstr "Límit de descàrrega assolit"

#: module/manager/PluginManager.py:153
#, python-format
msgid "%s has a invalid pattern."
msgstr "%s té un patró invàlid."

#: module/manager/PluginManager.py:272
#, python-format
msgid "Error importing %(name)s: %(msg)s"
msgstr "Error important %(name)s: %(msg)s"

#: module/plugins/internal/MultiHoster.py:133
msgid "No Hoster loaded"
msgstr "No s'ha carregat cap proveïdor"

#: module/plugins/account/BitshareCom.py:37
msgid "Activate direct Download in your Bitshare Account"
msgstr "Activa la descàrrega directa al teu compte de Bitshare"

#: module/plugins/container/LinkList.py:65
msgid "LinkList could not be cleared."
msgstr "LinkList no pot ser netejat."

#: module/manager/AccountManager.py:88
msgid "Account settings deleted, due to new config format."
msgstr "Els ajustaments del compte han estat eliminats degut al nou format de la configuració."

#: module/plugins/hoster/BasePlugin.py:65
msgid "Authorization required (username:password)"
msgstr "Autorització requerida (usuari:contrasenya)"

#: module/plugins/hoster/SimplydebridCom.py:25
#: module/plugins/hoster/RealdebridCom.py:41
#: module/plugins/hoster/FreeWayMe.py:38 module/plugins/hoster/ZeveraCom.py:22
#: module/plugins/hoster/UnrestrictLi.py:53
#: module/plugins/hoster/Premium4Me.py:28 module/plugins/hoster/FastixRu.py:36
#: module/plugins/hoster/AlldebridCom.py:38
#: module/plugins/hoster/DebridItaliaCom.py:40
#: module/plugins/hoster/RPNetBiz.py:26
#: module/plugins/hoster/MultiDebridCom.py:41
#: module/plugins/hoster/ReloadCc.py:24 module/plugins/hoster/RehostTo.py:26
#: module/plugins/hoster/PremiumizeMe.py:22
#: module/plugins/addon/RPNetBiz.py:43
#, python-format
msgid "Please enter your %s account or deactivate this plugin"
msgstr "Si us plau, introdueixi  el seu compte de %s o desactivar aquest plugin"

#: module/plugins/hoster/FilesMailRu.py:99
#, python-format
msgid "There was HTML Code in the Downloaded File (%s)...redirect error? The Download will be restarted."
msgstr "Hi havia Codi HTML en el fitxer (%s) descarregat... error de re-direcció? Es re-iniciarà la descàrrega."

#: module/plugins/hoster/NetloadIn.py:146
#: module/plugins/hoster/NetloadIn.py:170
msgid "File temporarily not available"
msgstr "Fitxer temporalment no disponible"

#: module/plugins/hoster/NetloadIn.py:183
#, python-format
msgid "Netload: waiting between downloads %d s."
msgstr "Netload: esperant entre descàrregues %d s."

#: module/plugins/hoster/NetloadIn.py:214
#, python-format
msgid "Netload: waiting for captcha %d s."
msgstr "Netload: esperant per captcha %d s."

#: module/plugins/hoster/NetloadIn.py:252
msgid "Downloaded File was empty"
msgstr "El fitxer descarregat estava buit"

#: module/plugins/hoster/UploadedTo.py:129
msgid "API key invalid"
msgstr "Clau de API invàlida"

#: module/plugins/hoster/UploadedTo.py:153
#, python-format
msgid "%s: Not enough traffic left"
msgstr "%s: No queda suficient tràfic"

#: module/plugins/hoster/UploadedTo.py:156
msgid "Traffic exceeded"
msgstr "Tràfic excedit"

#: module/plugins/hoster/RapidshareCom.py:100
msgid "Rapidshare: Traffic Share (direct download)"
msgstr "Rapidshare: Tràfic Compartit (descàrrega directa)"

#: module/plugins/hoster/RapidshareCom.py:127
#: module/plugins/hoster/RapidshareCom.py:194
msgid "Already downloading from this ip address, waiting 60 seconds"
msgstr "Ja s'està descarregant des de aquesta adreça IP, esperant 60 segons"

#: module/plugins/hoster/RapidshareCom.py:131
msgid "Invalid Auth Code, download will be restarted"
msgstr "Codi d'Autenticació invàlid, la descàrrega serà reiniciada"

#: module/plugins/hoster/RapidshareCom.py:199
msgid "RapidShareCom: No free slots"
msgstr "RapidShareCom: No hi ha espais lliures"

#: module/plugins/hoster/RapidshareCom.py:202
msgid "You need a premium account for this file"
msgstr "Necessites un compte premium per aquest fitxer"

#: module/plugins/hoster/RapidshareCom.py:204
msgid "Filename reported invalid"
msgstr "Nom de fitxer informat invàlid"

#: module/plugins/hoster/FileserveCom.py:99
msgid "Parallel download error, now waiting 60s."
msgstr "Error de descàrrega paral·lela, esperant 60s."

#: module/plugins/hoster/FileserveCom.py:215
msgid "Not logged in."
msgstr "No connectat."

#: module/plugins/hoster/MegaNz.py:56
msgid "Decryption failed"
msgstr "Desxifrat ha fallat"

#: module/plugins/hoster/MegaNz.py:106
msgid "No file key provided in the URL"
msgstr "Cap clau de fitxer proporcionat a la URL"

#: module/plugins/hoster/MegaNz.py:118
msgid "Error code:"
msgstr "Codi d'error:"

#: module/plugins/Container.py:68
msgid "File not exists."
msgstr "El fitxer no existeix."

#: module/plugins/addon/UpdateManager.py:74
msgid "*** Plugins have been updated, please restart pyLoad ***"
msgstr "* * * Plugins s'han actualitzat, si us plau reinicieu pyLoad * * *"

#: module/plugins/addon/UpdateManager.py:76
msgid "Plugins updated and reloaded"
msgstr "Plugins actualitzats i recarregats"

#: module/plugins/addon/UpdateManager.py:79
msgid "No plugin updates available"
msgstr "No hi ha actualitzacions de plugins disponibles"

#: module/plugins/addon/UpdateManager.py:96
msgid "No Updates for pyLoad"
msgstr "No hi ha actualitzacions del pyLoad"

#: module/plugins/addon/UpdateManager.py:100
#, python-format
msgid "***  New pyLoad Version %s available  ***"
msgstr "***  La nova Versió %s de pyLoad està disponible  ***"

#: module/plugins/addon/UpdateManager.py:101
msgid "***  Get it here: http://pyload.org/download  ***"
msgstr "***  Obtengui'l aquí: http://pyload.org/download  ***"

#: module/plugins/addon/UpdateManager.py:104
msgid "Not able to connect server for updates"
msgstr "No es pot connectar al servidor per actualitzacions"

#: module/plugins/addon/UpdateManager.py:148
#, python-format
msgid "New version of %(type)s|%(name)s : %(version).2f"
msgstr "Nova versió de %(type)s|%(name)s : %(version).2f"

#: module/plugins/addon/UpdateManager.py:157
#: module/plugins/addon/UpdateManager.py:162
#, python-format
msgid "Error when updating %s"
msgstr "S'ha produït un error mentre s'actualitzava %s"

#: module/plugins/addon/UpdateManager.py:162
msgid "Version mismatch"
msgstr "La versió no coincideix"

#: module/plugins/addon/IRCInterface.py:82
#, python-format
msgid "Download finished: %(name)s @ %(plugin)s "
msgstr "Descàrrega finalitzada: %(name)s @ %(plugin)s "

#: module/plugins/addon/IRCInterface.py:95
#, python-format
msgid "New Captcha Request: %s"
msgstr "Nova petició de Captcha: %s"

#: module/plugins/addon/IRCInterface.py:96
#, python-format
msgid "Answer with 'c %s text on the captcha'"
msgstr "Constesta amb 'c %s text del captcha'"

#: module/plugins/addon/Premium4Me.py:29
msgid "Please add your premium.to account first and restart pyLoad"
msgstr "Si us plau, afegeiu el vostre compte premium.to i torneu a iniciar pyLoad"

#: module/plugins/addon/HotFolder.py:82
#, python-format
msgid "Added %s from HotFolder"
msgstr "Afegit %s des de HotFolder"

#: module/plugins/addon/ExtractArchive.py:95
#, python-format
msgid "No %s installed"
msgstr "El %s no està instal·lat"

#: module/plugins/addon/ExtractArchive.py:97
#: module/plugins/addon/ExtractArchive.py:102
#, python-format
msgid "Could not activate %s"
msgstr "No s'ha pogut activar %s"

#: module/plugins/addon/ExtractArchive.py:107
msgid "Activated"
msgstr "Activat"

#: module/plugins/addon/ExtractArchive.py:109
msgid "No Extract plugins activated"
msgstr "No hi han connectors d'extracció activats"

#: module/plugins/addon/ExtractArchive.py:121
#, python-format
msgid "Package %s queued for later extracting"
msgstr "El paquet %s s'ha posat a la cua per una posterior extracció"

#: module/plugins/addon/ExtractArchive.py:144
#, python-format
msgid "Check package %s"
msgstr "Revisa el paquet %s"

#: module/plugins/addon/ExtractArchive.py:185
#, python-format
msgid "Extract to %s"
msgstr "Extreu a %s"

#: module/plugins/addon/ExtractArchive.py:200
msgid "No files found to extract"
msgstr "No s'han trobat fitxers per extreure"

#: module/plugins/addon/ExtractArchive.py:207
msgid "extracting"
msgstr "extraient"

#: module/plugins/addon/ExtractArchive.py:218
msgid "Password protected"
msgstr "Protegit amb contrasenya"

#: module/plugins/addon/ExtractArchive.py:239
msgid "Wrong password"
msgstr "Contrasenya incorrecta"

#: module/plugins/addon/ExtractArchive.py:247
#, python-format
msgid "Deleting %s files"
msgstr "Eliminant els fitxers %s"

#: module/plugins/addon/ExtractArchive.py:254
msgid "Extracting finished"
msgstr "Extracció finalitzada"

#: module/plugins/addon/ExtractArchive.py:260
msgid "Archive Error"
msgstr "Error d'arxiu"

#: module/plugins/addon/ExtractArchive.py:262
msgid "CRC Mismatch"
msgstr "El CRC no coincideix"

#: module/plugins/addon/ExtractArchive.py:266
msgid "Unknown Error"
msgstr "Error desconegut"

#: module/plugins/addon/ExtractArchive.py:318
msgid "Setting User and Group failed"
msgstr "Configuració d'usuari i el grup ha fallat"

#: module/plugins/addon/ClickAndLoad.py:75
msgid "Click'N'Load: Port 9666 already in use"
msgstr "Click'N'Load: El Port 9666 ja s'està en ús"

#: module/plugins/addon/CaptchaTrader.py:69
#: module/plugins/addon/Captcha9kw.py:59
#: module/plugins/addon/ExpertDecoders.py:50
#, python-format
msgid "%s credits left"
msgstr "%s crèdits restants"

#: module/plugins/addon/CaptchaTrader.py:117
msgid "Could not send response."
msgstr "No s'ha pogut enviar la resposta."

#: module/plugins/addon/CaptchaTrader.py:135
msgid "Your CaptchaTrader Account has not enough credits"
msgstr "El teu compte de CaptchaTrader no te crèdits suficients"

#: module/plugins/addon/LinkdecrypterCom.py:43
msgid "Crypter list not found"
msgstr "Llista de Crypter no trobada"

#: module/plugins/addon/LinkdecrypterCom.py:57
msgid "Crypter list is empty"
msgstr "La llista de Crypter està buida"

#: module/plugins/addon/XMPPInterface.py:91
#, python-format
msgid "Download finished: %(name)s @ %(plugin)s"
msgstr "Descàrrega finalitzada : %(name)s @ %(plugin)s"

#: module/plugins/addon/Captcha9kw.py:93
#, python-format
msgid "New CaptchaID from upload: %s : %s"
msgstr "Nou CaptchaID de càrrega: %s : %s"

#: module/plugins/addon/Captcha9kw.py:129
msgid "Your Captcha 9kw.eu Account has not enough credits"
msgstr "El teu compte de 9kw.eu no te crèdits suficients"

#: module/plugins/addon/ExternalScripts.py:54
#, python-format
msgid "Installed scripts for %s: "
msgstr "Scripts instal·lats per %s: "

#: module/plugins/addon/ExternalScripts.py:69
msgid "Script not executable:"
msgstr "Script no executable:"

#: module/plugins/addon/ExternalScripts.py:80
#, python-format
msgid "Error in %(script)s: %(error)s"
msgstr "Error en %(script)s: %(error)s"

#: module/plugins/addon/ExpertDecoders.py:95
msgid "Your ExpertDecoders Account has not enough credits"
msgstr "El teu compte de ExpertDecoders no te crèdits suficients"

#: module/plugins/addon/RehostTo.py:32
msgid "Please add your rehost.to account first and restart pyLoad"
msgstr "Si us plau afegir el seu compte de rehost.to primer i reinicieu pyLoad"

#: module/plugins/addon/PremiumizeMe.py:48
msgid "Please add a valid premiumize.me account first and restart pyLoad."
msgstr "Si us plau afegiu un compte vàlid premiumize.me primer i reinicieu pyLoad."

#: module/plugins/addon/CaptchaBrotherhood.py:69
#, python-format
msgid "%d credits left"
msgstr "%d crèdits restants"

#: module/plugins/Base.py:389
msgid "Pil and tesseract not installed and no Client connected for captcha decrypting"
msgstr "Pil i tesseract no estan instal·lats i no hi ha cap Client connectat per desxifrar captchas"

#: module/plugins/Base.py:393
msgid "No captcha result obtained in appropiate time by any of the plugins."
msgstr "Cap plugin ha pogut obtenir resultats apropiats d'aquest captcha en un temps assignat."

#: module/plugins/Base.py:498 module/plugins/Base.py:532
#, python-format
msgid "Setting User and Group failed: %s"
msgstr "Aplicació d'Usuari i el Grup ha fallat: %s"

#: module/manager/CaptchaManager.py:78
msgid "No Client connected for captcha decrypting"
msgstr "No hi ha cap client connectat per des-encriptar el captcha"

#: module/Api.py:330
#, python-format
msgid "Added package %(name)s containing %(count)d links"
msgstr "Afegit paquet %(name)s que conté les enllaços %(count)d"

#: module/Api.py:593
#, python-format
msgid "Added %(count)d links to package #%(package)d "
msgstr "Afegits %(count)d enllaços al paquet #%(package)d "

#: module/utils/JsEngine.py:156
msgid "No js engine detected, please install either Spidermonkey, ossp-js, pyv8 or rhino"
msgstr "No s'ha trobat cap motor js, si us plau instal·la Spidermonkey, ossp-js, pyv8 o rhino"

